acceptance/tests/options/config_file/no_custom_facts_and_custom_dir.rb

```diff
@@ -0,0 +1,43 @@
+# This test verifies that setting both custom-dir and no-custom-facts results in an error
+test_name "99994: config option no-custom-facts conflicts with custom-dir" do
+  tag 'risk:low'
+
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  content = <<EOM
+Facter.add('custom_fact') do
+  setcode do
+    "testvalue"
+  end
+end
+EOM
+
+  agents.each do |agent|
+    custom_dir = get_user_fact_dir(agent['platform'], on(agent, facter('kernelmajversion')).stdout.chomp.to_f)
+    on(agent, "mkdir -p '#{custom_dir}'")
+    custom_fact = File.join(custom_dir, 'custom_fact.rb')
+    create_remote_file(agent, custom_fact, content)
+
+    step "Agent #{agent}: create config file"
+    config_dir = agent.tmpdir("config_dir")
+    config_file = File.join(config_dir, "facter.conf")
+    config_content = <<EOM
+global : {
+    custom-dir : "#{custom_dir}"
+    no-custom-facts : true,
+}
+EOM
+    create_remote_file(agent, config_file, config_content)
+
+    teardown do
+      on(agent, "rm -rf '#{custom_fact}' '#{config_dir}'")
+    end
+
+    step "Agent #{agent}: config option no-custom-facts : true and custom-dir should result in an options conflict error" do
+      on(agent, facter("--config '#{config_file}'"), :acceptable_exit_codes => 1) do |facter_output|
+        assert_match(/options conflict/, facter_output.stderr.chomp, "Output does not contain error string")
```

Comment:
nit pick: There is no need to call `chomp` on `stderr` if you are performing a regex match on the string.